### PR TITLE
[Merged by Bors] - Fingerprint Lodestar

### PR DESCRIPTION
**Pull request by AgeManning**
Fingerprints the Lodestar client

### CODE REVIEW

File: beacon_node/eth2_libp2p/src/peer_manager/client.rs [hunk 1]
```diff
@@ -30,6 +30,8 @@ pub enum ClientKind {
     Teku,
     /// A Prysm node.
     Prysm,
+    /// A loadstar node.
+    Loadstar,
```
Comment (michaelsproul) [inline beacon_node/eth2_libp2p/src/peer_manager/client.rs:34]: *Lodestar

Comment (AgeManning) [reply beacon_node/eth2_libp2p/src/peer_manager/client.rs]: :sweat_smile:

File: beacon_node/eth2_libp2p/src/peer_manager/client.rs [hunk 2]
```diff
@@ -30,6 +30,8 @@ pub enum ClientKind {
     Teku,
     /// A Prysm node.
     Prysm,
+    /// A loadstar node.
```
Comment (michaelsproul) [inline beacon_node/eth2_libp2p/src/peer_manager/client.rs:33]: One more :joy:

Comment (AgeManning) [reply beacon_node/eth2_libp2p/src/peer_manager/client.rs]: Damn. You're on it! Eagle eye Michael. 

Thanks :blush:

**Pull request closed without merging**